src/components/DeviceStatusGauge.js

```diff
@@ -0,0 +1,241 @@
+import { h } from 'preact'
+import _ from 'lodash'
+import Pie from 'paths-js/pie'
+import styled, { withTheme } from 'styled-components'
+import { compose } from 'recompose'
+import ResinDeviceStatus from 'resin-device-status'
+
+import hoc from '../hoc'
+import theme from '../theme'
+
+const { colors, font } = theme
+
+const DeviceStatusGaugeWrapper = styled.div`
+  display: block;
+
+  .device-status-gauge__graph {
+    display: block;
+    width: 100%;
+    max-width: 180px;
+    margin: 0 auto 10px;
+  }
+
+  .device-status-gauge__graph__total-devices-label {
+    font-family: ${font};
+    font-size: 5px;
+    fill: #b8b8b8;
+    text-anchor: middle;
+  }
+
+  .device-status-gauge__graph__total-devices-number {
+    font-family: ${font};
+    font-size: 26px;
+    fill: #5e5e5e;
```

Comment:
Should these colors should be derived from our theme?

Reply:
Yep that would be better